### PR TITLE
feat: wishlist

### PR DESCRIPTION
**Pull request by shivangisharma01**
Wishlist
image

### CODE REVIEW

File: Page/Wishlist/wishlist.css [hunk 1]
```diff
@@ -0,0 +1,19 @@
+.wishlist-container-box{
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
```
Comment (rittik-ghosh) [inline Page/Wishlist/wishlist.css]: can use the common properties in utility class

**Review by rittik-ghosh — COMMENTED** [review]
The code is clear and readable. Good job!!.
Have suggested some changes for later.

**Review by nishasen — APPROVED** [review]
Please loog into the comment of ritik, else everything looks good.

File: Page/Wishlist/wishlist.css (hunk 1 above)
Comment (shivangisharma01) [reply Page/Wishlist/wishlist.css]: updated

**Pull request merged**